src/style.css

```
@@ -1 +1,80 @@
-/* Styles go here */
+body {
+  font-family: "Robota", sans-serif;
+}
+
+*{
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__div {
+  margin-right: 50px;
+  display: flex;
+  justify-content: center;
+ }
+
+ .image {
+  display: flex;
+  margin-left: 50px;
+ }
+
+.nav__lint {
+  margin-right: 50px;
+  display: flex;
+  justify-content: center;
+}
+
+.nav__item {
+  padding-left: 0;
+  display: flex;
+  justify-content: flex-end;
+  list-style: none;
+  text-transform: uppercase;
+}
+
+.nav__li {
+  margin-right: 20px;
+}
+
+.nav__link {
+    display: block;
+    padding: 23px 0;
+    text-decoration: none;
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 14px;
+    text-transform: uppercase;
+    color: #000;
+}
+
+.nav__link:hover {
+    color: 00acdc;
```

```suggestion
    color: #00acdc;
```
but move colors you use more than once into CSS variables